### PR TITLE
Add Better Uptime

### PR DESCRIPTION
**Pull request by Leigh-Ola**

### CODE REVIEW

**Comment by Leigh-Ola** [comment]
Issue may stem from the fact that I'm trying to add betteruptime.com, and BetterUptime is available as a feature offered by betterstack.com

File: img/b/betterstack.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...1.992 1.134-3.309 2.91-3.309 1.655 0 2.697 1.317 2.697 3.309v8.762h4.289v-8.762c0-1.992 1.133-3.309 2.91-3.309 1.655 0 2.727 1.317 2.727 3.309v8.762h4.258v-10.11c0-3.095-2.175-5.576-5.545-5.576-2.481 0-4.412 1.317-5.147 2.819-.735-1.839-2.298-2.819-4.718-2.819-2.083 0-3.554 1.011-4.381 2.42v-2.175h-4.289v15.441ZM244.678 30.867c-.337 1.317-1.471 1.991-3.186 1.991-2.145 0-3.524-1.347-3.83-3.523h11.09c.031-.337.062-.796.062-1.378 0-3.524-2.176-7.537-7.629-7.537-5.239 0-7.628 4.013-7.628 7.935 0 3.89 2.696 7.996 7.935 7.996 3.798 0 6.74-2.144 7.352-5.484h-4.166Zm-3.493-7.322c2.053 0 3.248 1.379 3.37 3.094h-6.832c.368-2.144 1.532-3.094 3.462-3.094Z"/><path fill="url(#a)" d="M22.455 24.573V8.629c0-2.884-3.623-4.17-5.442-1.932L4.06 22.641c-1.627 2.002-.202 4.995 2.378 4.995H19.39a3.064 3.064 0 0 0 3.064-3.063Z"/><path fill="#fff" d="M31.192 22.455H18.788a3.241 3.241 0 0 0-3.241 3.241v15.265c0 3.052 3.832 4.413 5.757 2.044L33.707 27.74c1.721-2.118.214-5.285-2.515-5.285Z"/><defs><linearGradient id="a" x1="10.364" x2="25.046" y1="6.045" y2="33.682" gradientUnits="userSpaceOnUse"><stop stop-color="#BCBECD"/><stop offset="1" stop-color="#BCBECD" stop-opacity="0"/></linearGradient></defs></svg>
```
Comment (Carlgo11) [inline img/b/betterstack.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 31.07 38.65"><defs><linearGradient id="a" x1="10.36" x2="25.05" y1="6.05" y2="33.68" gradientTransform="translate(-3.37 -5.56)" gradientUnits="userSpaceOnUse"><stop stop-color="#BCBECD"/><stop offset="1" stop-color="#BCBECD" stop-opacity="0"/></linearGradient></defs><path fill="url(#a)" d="M19.09 19.01V3.07a3.06 3.06 0 0 0-5.44-1.93L.69 17.08c-1.63 2-.2 5 2.38 5h12.95c1.7 0 3.07-1.37 3.07-3.07z"/><path fill="#fff" d="M27.82 16.9h-12.4a3.24 3.24 0 0 0-3.24 3.24V35.4a3.24 3.24 0 0 0 5.76 2.05l12.4-15.27a3.24 3.24 0 0 0-2.52-5.28z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Hello @Leigh-Ola,
apologies for the delay in reviewing your PR.

I see you attempted to fix a domain name error. The best way of doing this is to rename the file `betteruptime.com.json` and change the domain element to `betteruptime.com`. You can then use the url element to direct people to `https://betterstack.com/better-uptime`.

You entered the Developer tools and Domain Name registration categories in the keywords array. Is this correct?

Regarding the 2FA methods, as the documentation doesn't mention anything about which methods are supported we need some proof that TOTP in fact is supported. Please attach some screenshots of the 2FA setup process and/or login process.
> **Note** Remember to blur or crop out any personal information linked to you.

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by Carlgo11** [comment]
Closing due to inactivity.
> **Note**
> The PR author you can still commit the requested changes and re-open this PR.
> If someone else wants to commit the requested changes you can create a new PR and link to this PR.

**Pull request closed without merging**